meta/autoid/autoid_test.go

```diff
@@ -22,405 +22,395 @@ import (
 	"testing"
 	"time"
 
-	. "github.com/pingcap/check"
 	"github.com/pingcap/errors"
 	"github.com/pingcap/failpoint"
 	"github.com/pingcap/parser/model"
 	"github.com/pingcap/tidb/kv"
 	"github.com/pingcap/tidb/meta"
 	"github.com/pingcap/tidb/meta/autoid"
 	"github.com/pingcap/tidb/store/mockstore"
+	"github.com/stretchr/testify/require"
 )
 
 func TestT(t *testing.T) {
```

Comment:
You should find a better name instead of `TestT`. And may try to separate tests as small as possible, may in different files.